File: Java/CodingBat/noTeenSum.java
// http://codingbat.com/prob/p182879

public int noTeenSum(int a, int b, int c) {
  return fixTeen(a) + fixTeen(b) + fixTeen(c);
}

public int fixTeen(int n) {
  return (n >= 13 && n <= 19 && n != 15 && n != 16) ? 0 : n;
}

File: Python/CodingBat/no_teen_sum.py
# http://codingbat.com/prob/p100347

def no_teen_sum(a, b, c):
  return fix_teen(a) + fix_teen(b) + fix_teen(c)

def fix_teen(n):
  if n in range(13, 20):
    if n == 15:
      return 15
    elif n == 16:
      return 16
    else:
      return 0
  else:
    return n
